system-tests/_scripts/launch-cluster.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if [ -z "${INSTALLER_URL}" ]; then
+  echo "Error: Please specify the INSTALLER_URL environment variable"
+  exit 1
+fi
+
+if [ -z  "${SSH_KEY_PATH}" ]; then
+  echo "Error: Please specify the SSH_KEY_PATH environment variable"
+  exit 1
+fi
+
+# Create cluster config
+cat << EOF > config.yaml
+---
+launch_config_version: 1
+deployment_name: dcos-ui-system-test-$(date +%s)
+installer_url: ${INSTALLER_URL}
+platform: aws
+provider: onprem
+aws_region: us-west-2
+aws_key_name: default
+ssh_private_key_filename: ${SSH_KEY_PATH}
+os_name: cent-os-7
+instance_type: m4.large
+num_masters: 1
+num_private_agents: 1
+num_public_agents: 1
+dcos_config:
+    cluster_name: DC/OS UI System Integration Tests
+EOF
+
+if ! dcos-launch create; then
+  echo "Error: Cluster launch failed."
+  exit 1
+fi
+if ! dcos-launch wait; then
```

Comment:
If you implement what I mentioned above you will now need to specify the cluster info file (`-i $CLUSTER_INFO`)